docs/product/issues/issue-details/breadcrumbs/index.mdx

```diff
@@ -1,61 +1,45 @@
+- **Sort**: Determines how breadcrumbs will be ordered, either Newest first or Oldest first. Your sort preference is saved to the browser.
+- **Time Display** Determines how breadcrumb timestamps will be displayed, either as absolute timestamps (respecting user preferences for 24-hour clocks/timezone), or relative to the first breadcrumb in the list. Relative timestamps that appear negative indicate they took place before the event first breadcrumb. Your time display preference is saved to the browser.
 
-A semi-internal attribute `type` can control the type of the breadcrumb. By default, all breadcrumbs are recorded as `default`, which makes them appear as a `Debug` entry, but Sentry provides other [types](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types) that influence how the breadcrumbs are rendered.
+![Breadcrumbs drawer](./img/breadcrumbs-drawer.png)
 
-Category
 
-The category of the event. This data is similar to a logger name, and helps you understand the area in which an event took place, such as auth.
+## Breadcrumbs Attributes
 
-Description
+Each breadcrumb is composed of the following details:
 
-A description is either:
+- **Type**: A semi-internal attribute `type` can control the type of the breadcrumb. By default, all breadcrumbs are recorded as `default`, but Sentry provides other [types](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types) that influence the color/icon associated with the breadcrumb in the UI.
 
-- Message: A string describing the event, rendered as text with all whitespace preserved. It's often used as a drop-in for a traditional log message.
+- **Category**: The category of the event. This data is similar to a logger name, and helps you understand the area in which an event took place, such as `auth`. In the UI, period characters are replaced with space (e.g. `ui.click` becomes `UI Click`)
 
-- Data: A key-value mapping of event metadata. This is often used instead of a message, but may be used in addition to one. [sentry.io](https://sentry.io) displays all the data sent.
+- **Message**: A string describing the event, rendered as text with all whitespace preserved. It's often used as a drop-in for a traditional log message.
 
-Level
+- **Data**: A key-value mapping of breadcrumb's arbitrary data. Useful for attaching structured information related to the breadcrumb, like IDs or variable values.
 
-The severity of an event. The level is set to one of five values, which are — in order of severity — fatal, error, warning, info, and debug.
+- **Level**: The severity of an event. The level is set to one of five values, which are — in order of severity — fatal, error, warning, info, and debug.
 
-Time
+- **Time**: A timestamp representing when the breadcrumb occurred. The format is either a string, as defined in [RFC 3339](https://tools.ietf.org/html/rfc3339), or a numeric value representing the number of seconds that have elapsed since the Unix epoch.
```

Comment:
```suggestion
- **Time**: A timestamp showing when the breadcrumb occurred. The format is either a string, as defined in [RFC 3339](https://tools.ietf.org/html/rfc3339), or a numeric value representing the number of seconds that have elapsed since the Unix epoch.
```